src/index.js

```diff
@@ -92,13 +100,17 @@ module.exports = async function App(config) {
   const shouldNotPostToPr = config.get('github:skip_pr_post');
   if (shouldNotPostToPr) return;
 
-  await delaySending(
-    postToPRHandler,
-    prNumbers.map(prNumber => ({
-      head,
-      owner,
-      repo,
-      number: prNumber,
-    })),
-  );
+  try {
+    await delaySending(
+      postToPRHandler,
+      prNumbers.map(prNumber => ({
+        head,
+        owner,
+        repo,
+        number: prNumber,
+      })),
+    );
+  } catch (error) {
+    console.log(`Unable to post back to pull requests with release notes: ${error}`);
```

Comment:
Would you prefer to print to stderr instead of stdout? And dump the full error object so we get a stack trace?

Unless this is meant as user messaging and the stack is intentionally suppressed?

```suggestion
    console.error('Unable to post back to pull requests with release notes', error);
```